<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWeatherComponent</class>
 <widget class="QWidget" name="CWeatherComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <property name="currentIndex" stdset="0">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_weatherControl">
     <property name="title">
      <string>Weather Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="cb_weatherScenario"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_weatherDisplay">
     <property name="title">
      <string>Weather Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cb_UseOwnAcftPosition">
        <property name="text">
         <string>Use Own Aircraft Position</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="le_LatOrIcao">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Lat or ICAO</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_Lon">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Lon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lb_Status">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QTabWidget" name="tw_weatherGrid">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tb_temperatureLayers">
         <attribute name="title">
          <string>Temperature</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CTemperatureLayerView" name="tvp_TemperatureLayers"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tb_cloudLayers">
         <attribute name="title">
          <string>Clouds</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CCloudLayerView" name="tvp_CloudLayers"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tb_windLayers">
         <attribute name="title">
          <string>Winds</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CWindLayerView" name="tvp_WindLayers"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CTemperatureLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/temperaturelayerview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CCloudLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/cloudlayerview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CWindLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/windlayerview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
